forgettable-frontend/src/components/InputField/InputField.module.css

```diff
@@ -0,0 +1,55 @@
+.inputLabelPrimary {
+    font-family: 'Poppins', sans-serif;
+    font-size: 23px;
+    font-weight: bold;
```

Comment:
Could we use the css vars from index.css instead of hard coding fontsize fontweight? it would look something like:
`font-size: var(--text-large)`
This just helps to keep the theme consistent if we were to change linked details :) You can look at some of the other css modules if you need an example!

Reply:
I have covered the font sizes, font weights, and corner radii to use global CSS styles as seen in "index.css". Let me know if I have missed any. Thanks.